src/server.c

```diff
@@ -1736,27 +1736,61 @@ int clientsCronTrackExpansiveClients(client *c, int time_idx) {
     return 0; /* This function never terminates the client. */
 }
 
-/* Iterating all the clients in getMemoryOverheadData() is too slow and
- * in turn would make the INFO command too slow. So we perform this
- * computation incrementally and track the (not instantaneous but updated
- * to the second) total memory used by clients using clinetsCron() in
- * a more incremental way (depending on server.hz). */
-int clientsCronTrackClientsMemUsage(client *c) {
-    size_t mem = 0;
+clientMemUsageBucket *getMemUsageBucket(size_t mem) {
```

Comment:
this function is missing some top comment to describe the buckets mechanism.
maybe just something short with a reference to the doc in server.h

Reply:
done.